Technological_Sovereignty_Green_Paper_1.md

```diff
@@ -319,7 +320,7 @@ recognized:
 67) - Right of equal treatment: Citizens have the right to be free from algorithmic discrimination.
      - If algorithmic services provide outputs of consistently lower value or quality to or about users coming from historically marginalized backgrounds, this constitutes discrimination ([example from google search](http://time.com/5209144/google-search-engine-algorithm-bias-racism/)).
      - Users should be able to compare outputs based on different demographic profiles (e.g. "would this search result be the same if I were to change the gender or age the algorithm has inferred for me?").
```

Comment:
Feedback from [F12](https://github.com/kmccurdy/techdsc/wiki/F12-e-mail-by-Benjamin-K.):

> **67.2 should be checked with actual AI/ML people **- the way I understand it, those systems often act like a black box and can't necessarily be "understood" (as much as human brain can't be "understood"); and for many different aspects expecting demographic data to be interchangeable doesn't only render them useless, it also inhibits the possibility for them to make better decisions - just as an example, not knowing a patients sex inhibits vastly how good the treatment after a heart attack is, women - over all - showing very different symptoms. While honorable in thought, the idea of "equal"-treatment build here, only benefits the masses and lets us miss the opportunities that mass computing bring

Reply:
two ways I can think of to handle this (with some commentary):
- elaborate on the domains in which this right would be applied 
  - e.g. higher-stakes contexts (medical) vs. lower/unclear stakes (web search - although the stakes here can vary a great deal, from nonexistent (looking up a fact) to consequential (searching for job opportunities, sensitive personal or medical advice, ...))
  - as someone in AI/ML, I would say that, for even black box algorithms, one can change the feature space to compare outcomes - this would be one of the key ways of conducting audits
  - also: it's not about *denying* people access to personalized outcomes, but being able to justify them: if my gender has been an outcome in helping my doctor determine proper treatment, I'm not opposed to that, but it's a context in which the decision can be justified (importantly, there's a human interface - if I want to know *why* I got a particular treatment, I can ask my doctor, and s/he will presumably be able to give information *beyond* "the computer said to do this") - but in a web search, if outcomes have been affected my my (given or inferred) gender data, I can still demand the reasoning behind it, and attempt to see what the outcome would be like otherwise
- or: just drop, as it's covered as well in the following section